docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -134,6 +200,61 @@ Displays the help and exits.
 -h, --help
 ```
 
+### tls-allow-any-client
+
+Allows any client that presents a certificate to connect.
```

Comment:
@rain-on please re-confirm. I think with `tls-allow-any-client`, the client may not have to present any certificate at all.

Reply:
Done.